src/components/PanelSidetray/PanelSidetray.vue

```diff
@@ -0,0 +1,112 @@
+<template lang="html">
+    <div class="panel-sidetray">
+        <nuxt-link
+            v-for="(item, i) in items"
+            v-if="item.featuredImage.sourceUrl"
+            :key="item.id"
+            :to="item.uri"
+            @click.native="handleClick"
```

Comment:
Don't need this click event. I'm going to make any click on the entire component close the panel.